lib/WeBWorK/Utils/FormatRecords.pm

```diff
@@ -115,9 +115,9 @@ use constant PRESET_FORMATS => {
 				field_order     => [qw/assignment_type set_id due_date/],
 				format_function => sub {
 					join('',
-						FIELD_PROPERTIES()->{assignment_type}{labels}{ $_[0] },
-						': ', format_set_name_display($_[1]),
-						', ', formatDateTime($_[2]));
+						FIELD_PROPERTIES()->{assignment_type}{labels}{ $_[1] },
+						': ', format_set_name_display($_[2]),
+						', ', $_[0]->formatDateTime($_[3]));
```

Comment:
Note that these `WeBWorK::ContentGenerator::formatDateTime` calls use the defaults for the optional 2nd, 3rd, and 4th parameters.  Those will change to better values for this with pull request openwebwork/webwork2#2263.